docs/en/commands/status.md

```diff
@@ -77,20 +90,22 @@ Implementations may optionally include additional meta information as part of th
 
 ### URL Parameters
 
-None
+|name|description|
```

Comment:
probably don't want a blank table, right?

Reply:
Right. I need to move the if block.